templates/cloud-provider-azure/cloud-node-manager.yaml

```diff
@@ -0,0 +1,85 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    k8s-app: cloud-node-manager
+  name: cloud-node-manager
+  namespace: kube-system
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: cloud-node-manager
+  labels:
+    k8s-app: cloud-node-manager
+rules:
+  - apiGroups: [""]
+    resources: ["nodes"]
+    verbs: ["watch", "list", "get", "update", "patch"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: cloud-node-manager
+  labels:
+    k8s-app: cloud-node-manager
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cloud-node-manager
+subjects:
+  - kind: ServiceAccount
+    name: cloud-node-manager
+    namespace: kube-system
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: cloud-node-manager
```

Comment:
This cloud-node-manager doesn't include the windows DS.  Is windows working without it?

The windows deamon set is provided in the examples: https://github.com/kubernetes-sigs/cloud-provider-azure/blob/66e1bec130df801212310a14bedae737fefd042e/examples/out-of-tree/cloud-node-manager.yaml#L102-L108)

I think it will be required for all of the Windows functionality to work.  @pengfi might know?

Reply:
No, windows should also have a daemonset

Reply:
I think this was missed because our out tree cloud provider test doesn't run against windows nodes: https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/2209#discussion_r840897562

https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/c55fe5509ab16e191978b570084b1605e6a05217/test/e2e/azure_test.go#L434